_pkgdown.yml

```diff
@@ -25,8 +25,7 @@ reference:
   - has_keyword("der_adsl")
 - subtitle: BDS-specific
   desc: Derivation Functions helpful for building the BDS datasets (e.g. advs, adlb, adeg, adex)
-- contents:
-  - has_keyword("der_bds_gen")
+  contents:
```

Comment:
why is there now no `-` before contents?

Reply:
It doesn't matter if there is a `-` before `contents:` or not. I have removed it when experimenting.

Reply:
i would say best to include it regardless for consistency with the other lines

Reply:
Ok, I have made it consistent. I have removed the `-` before `contents:` because I think it is better readable and it is in line with the tidyverse packages.